pkg/executor/sortexec/sort_spill_test.go

```diff
@@ -99,6 +99,8 @@ func (r *resultChecker) initRowPtrs() {
 }
 
 func (r *resultChecker) check(resultChunks []*chunk.Chunk, offset int64, count int64) bool {
+	ctx := mock.NewContext()
```

Comment:
you can also use `StaticEvalContext` here

Reply:
Good! I replaced most new `mock.NewContext()` in this PR into `contextstatic.NewStaticEvalContext()` :+1:.